thorlcr/activities/diskread/thdiskread.cpp

```diff
@@ -256,6 +259,18 @@ class CDiskGroupAggregateActivityMaster : public CDiskReadMasterVF
         if (!container.queryLocalOrGrouped())
             dst.append(mpTag);
     }
+    void deserializeActivityStats(unsigned node, MemoryBuffer &mb)
+    {
+        CDiskReadMasterVF::deserializeActivityStats(node, mb);
+        unsigned __int64 blockedNs;
+        mb.read(blockedNs);
+        blockedTime->set(node, blockedNs);
+    }
+    virtual void getActivityStats(IStatisticGatherer & stats)
+    {
+        CDiskReadMasterVF::getActivityStats(stats);
+        blockedTime->getStats(stats,false);
```

Comment:
my expectation was that blocked was a universal property of all activities, so would be serialized and deserialized for each, so this would be in the base class.

It's true that not all activities can block, probably about half of them can though..
_could_ have a single byte to indicate whether act. has accumulated any blocked time or not (to save other 7 bytes per act. that doesn't send.), but doubt worth it.